spec/presenters/contact_presenter_spec.rb

```diff
@@ -23,13 +23,15 @@
     expect(payload[:need_ids]).to be_empty
 
     details = payload[:details]
-    expect(details[:slug]).to eq(contact.slug)
-    expect(details[:title]).to eq(contact.title)
```

Comment:
I was about to respond "no we don't, content-store organisations (as expanded from links) don't have slugs", but then I remembered that the step of _only_ including them in the links hash is a thing we're doing later. Reinstated.